source/_patterns/02-molecules/vue-widget/demo/index.js

```diff
@@ -0,0 +1,36 @@
+import Vue from 'vue';
+
+// Demo twig
+import twig from './vue-widgets.twig';
```

Comment:
Curious about this. Will Twig have rendered html by the time the Vue widget picks it up here? e.g., say we had a Twig markup for `<h1>{{ kittens }}</h1>` where `kittens` had a value of `Scottish Fold`, would we see an H1 for Scottish Fold or an empty space?